examples/index.html

```diff
@@ -5,7 +5,7 @@
   <title>check-all demo</title>
 </head>
 <body>
-  <div data-check-all-container>
+  <div data-check-all-container style="user-select: none; -webkit-user-select: none;">
```

Comment:
If there's a selection, shift-click would be used to modify selection in Chrome. This sets user-select to none for a more consistent shift-click behavior. In reality this should probably only get scope to the label elements, or not at all.